### ISSUE TITLE
i have optifabric error. pls help me

### ISSUE THREAD

**Issue by yoviks** [Chocohead/OptiFabric#1708]
![photo_5472174298501340211_y](https://github.com/Chocohead/OptiFabric/assets/174058350/ade978b3-f7e3-44c8-b7dd-3bddc0270b3c)

**Comment by Dracconis** [comment]
Use Fabric Loader 0.14.25, also next time please do a search to see if your issue has already been solved. If this solves your issue please close the ticket.

**Comment by yoviks** [comment]
well if i use a Tlauncher this advice also help me?

**Comment by Dracconis** [comment]
> well if i use a Tlauncher this advice also help me?

Yes. Follow here: https://tlauncher.org/en/how-install-fabric.html
